templates/zerver/help/view-and-edit-your-message-drafts.md

```diff
@@ -0,0 +1,72 @@
+# View and edit your message drafts
+
+Message drafts are created when unsent messages are discarded by exiting out
+of the compose box while content has been written in the **Compose your message here...**
+field. Zulip's [Drafts](/#drafts) feature allows users to retrieve and edit
+draft contents in order to send them as new messages.
+
+## View your drafts
+
+To view a list of all your currently saved drafts, you can open the
+[Drafts](/#drafts) modal by clicking the **Drafts** button on the left side of
+the compose box located at the bottom of your window or pressing the `d` key.
+Upon doing so, the **Drafts** modal will appear, displaying all of the drafts
+you currently have in the order that they were created.
+
+![Drafts modal](/static/images/help/drafts-modal.png)
+
+Each draft created from a discarded stream message is displayed with the
+following information:
+
+* **Stream** — If the discarded message that the draft was created from had a
+specified stream, the draft will display the message's specified stream in the
+top-left corner.
+
+* **Topic** — If the discarded message that the draft was created from had a
+specified topic, the draft will display the message's topic to the right of the
+message's stream.
+
+* **Content** — The content of the discarded message that the draft was created
+from is displayed as the draft's content.
+
+If a draft was created from a discarded private message instead of a stream
+message, the private message's recipient(s) will be displayed in place of the
+stream and topic.
+
+!!! tip ""
+    There are various [keyboard shortcuts](/help/keyboard-shortcuts#drafts) that
+    allow you to easily navigate the **Drafts** modal.
+
+## Edit a draft
+
+To edit the content of a draft, click the pencil (<i
+class="icon-vector-pencil"></i>) icon or press the `Enter` key to open the
+selected draft in the compose box, where you can change the stream and topic
+(for stream messages), recipient (for private messages), and content of the
+draft.
+
+To save the changes you made to the draft, exit out of the compose box by
+clicking the x (<i class="icon-vector-remove"></i>) or pressing the `Esc` key.
+
+!!! warn ""
+    **Note:** Drafts cannot be converted from one message type to another
+    (stream to private message and vice versa). You will have to discard the
+    draft and create a new draft with the appropriate message type if you wish
+    to do so.
+
+## Send a draft
+
+Once you are ready to send a draft as a message, click the pencil (<i
+class="icon-vector-pencil"></i>) icon or press the `Enter` key to open the
+selected draft in the compose box. You can now send the draft as a message by
+pressing the `Enter` key or clicking the **Send** button, depending on your
+settings.
```

Comment:
There doesn't seem to be any error messages that appear when saving drafts with invalid content.